cmd/cli/devstack/devstack.go

```diff
@@ -166,17 +166,17 @@ func runDevstack(cmd *cobra.Command, ODs *devstack.DevStackOptions, IsNoop bool)
 
 		// If we don't set the repo value in config, readers will be given
 		// a different path to the one we've just created. Presumably a default.
-		config.SetValue("repo", repoPath)
+		cfg.System().Set("repo", repoPath)
 		defer os.RemoveAll(repoPath)
 	}
 
-	fsRepo, err := setup.SetupBacalhauRepo(repoPath)
+	fsRepo, err := setup.SetupBacalhauRepo(repoPath, cfg)
 	if err != nil {
 		return err
 	}
 
 	// make sure we don't run devstack with a custom IPFS path - that must be used only with serve
-	if path, err := config.Get[string](types.NodeIPFSServePath); err == nil && path != "" {
+	if path, err := config.Get[string](cfg, types.NodeIPFSServePath); err == nil && path != "" {
```

Comment:
Because methods cannot have type parameters (generics): https://go.googlesource.com/proposal/+/refs/heads/master/design/43651-type-parameters.md#no-parameterized-methods